docs/esp32/quickref.rst

```diff
@@ -538,3 +538,31 @@ the corresponding functions, or you can use the command-line client
 
 See the MicroPython forum for other community-supported alternatives
 to transfer files to an ESP32 board.
+
+Interrupts
+----------
+**Hardware Interrupts** - They are fired by hardware peripherals and sources.hey can be: Internal or External. An example of internal hardware interrupts maybe something like a hardware timer interrupt or WDT. An example of an external hardware interrupt is external GPIO pins interrupts 
```

Comment:
Small typo - "sources.hey" should be "sources. They"

Reply:
Looks good to me - I'm not a maintainer so can't merge things unfortunately.

Reply:
@dpgeorge Please see if it can be merged